tests/countries/test_algeria.py

```diff
@@ -12,161 +12,341 @@
+    def test_ascension_day(self):
+        name = "عيد الصعود"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name,
+            self.christian_holidays,
+            "2020-05-21",
+            "2021-05-13",
+            "2022-05-26",
+            "2023-05-18",
+            "2024-05-09",
+            "2025-05-29",
+        )
+        self.assertHolidayName(name, self.christian_holidays, range(1964, 2050))
 
-        # Mawlid / Prophet's Birthday
-        self.assertNoHoliday(
-            "2021-10-19",
-            "2023-09-28",
+    def test_whit_monday(self):
+        name = "إثنين العنصرة"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name,
+            self.christian_holidays,
+            "2020-06-01",
+            "2021-05-24",
+            "2022-06-06",
+            "2023-05-29",
+            "2024-05-20",
+            "2025-06-09",
         )
-        self.assertHoliday(
-            "2021-10-18",
-            "2023-09-27",
+        self.assertHolidayName(name, self.christian_holidays, range(1964, 2050))
+
+    def test_assumption_day(self):
+        name = "عيد انتقال السيدة العذراء"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name, self.christian_holidays, (f"{year}-08-15" for year in range(1964, 2050))
+        )
+
+    def test_christmas_day(self):
+        name = "عيد الميلاد"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name, self.christian_holidays, (f"{year}-12-25" for year in range(1964, 2050))
+        )
+
+    def test_rosh_hashanah(self):
+        name = "رأس السنة العبرية"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name,
+            self.hebrew_holidays,
+            "2020-09-19",
+            "2021-09-07",
+            "2022-09-26",
+            "2023-09-16",
+            "2024-10-03",
+            "2025-09-23",
+        )
+        self.assertHolidayName(name, self.hebrew_holidays, range(1964, 2050))
+
+    def test_yom_kippur(self):
+        name = "يوم الغفران"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name,
+            self.hebrew_holidays,
+            "2020-09-28",
+            "2021-09-16",
+            "2022-10-05",
+            "2023-09-25",
+            "2024-10-12",
+            "2025-10-02",
+        )
+        self.assertHolidayName(name, self.hebrew_holidays, range(1964, 2050))
+
+    def test_pesach(self):
+        name = "عيد الفصح اليهودي"
+        self.assertNoHolidayName(name)
+        self.assertHolidayName(
+            name,
+            self.hebrew_holidays,
+            "2020-04-09",
+            "2021-03-28",
+            "2022-04-16",
+            "2023-04-06",
+            "2024-04-23",
+            "2025-04-13",
         )
+        self.assertHolidayName(name, self.hebrew_holidays, range(1964, 2050))
 
     def test_l10_default(self):
```

Comment:
```suggestion
    def test_l10n_default(self):
```
I didn't notice it at first. 🤷‍♂️ (Same for Sri Lanka, will need to be fixed separately)